packages/shorebird_code_push_client/lib/src/version.dart

```diff
@@ -1,2 +1,2 @@
 // Generated code. Do not modify.
-const packageVersion = '0.1.0+1';
+const packageVersion = '0.2.0+1';
```

Comment:
Should this be in a separate commit?

Reply:
figured I'd make it here since this is a breaking change and I don't want to forget